main-site/components/testimonials-section/testimonial-card/TestimonialCard.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { TestimonialCardProps } from '../../../lib/types';
+import {
+  StyledTestimonialCardContainer,
+  StyledTestimonialCardBox,
+  StyledTestimonialCardAuthor,
+  StyledTestimonialCardImage,
+  StyledTestimonialCardQuote,
+  StyledTestimonialCardYear
+} from './TestimonialCard.styles';
+
+import Karyna from '../../../../shared-ui/images/karyna.png';
+import Jimin from '../../../../shared-ui/images/jimin.png';
+import Spencer from '../../../../shared-ui/images/spencer.png';
+import Group from '../../../../shared-ui/images/group.png';
+
+function getImage(author: string): string {
+  if (author === 'Karyna Yen') {
+    return Karyna;
+  }
+  if (author === 'Ji-min Kim') {
+    return Jimin;
+  }
+  if (author === 'Spencer Shao') {
+    return Spencer;
+  }
+  return Group;
+}
+
+const TestimonialCard: React.FC<TestimonialCardProps> = ({
+  id,
+  author,
+  year,
+  quote,
+  currentIndex
+}) => {
+  const isSelected: boolean = currentIndex == id;
+  return (
+    isSelected && (
+      <StyledTestimonialCardContainer>
+        <StyledTestimonialCardBox>
+          <StyledTestimonialCardImage src={getImage(author)} />
+          <StyledTestimonialCardAuthor>
+            <p>{author}</p>
```

Comment:
dumb question because i've been in the app portal for too long: is there a custom styled <P> tag that we should be using instead?

Reply:
Yes you are right! I use a custom tag in `StyledTestimonialCardAuthor` and forgot to remove the  `<p>`. Nice catch!